### PR TITLE
Candidate for release_v4.11.0

### PR DESCRIPTION
**Pull request by govindrao55**

### CODE REVIEW

File: examples/analytics/main.tf [hunk 1]
```diff
@@ -51,7 +49,7 @@ resource "oci_analytics_analytics_instance" "test_oce_instance_public" {
...
...IjoiTE9DQUwiLCJ0ZW5hbnQiOiJpZGNzLTlmZWIzZWY5MWNhNDQ0MWJiZWRhZGJjNTY0MGQ1ZmI4IiwianRpIjoiYjAyNzJiYWItM2Y1MC00YTMzLWJhZjYtN2RmMjg4MzdiNDRkIiwidXNlcl9kaXNwbGF5bmFtZSI6IlBhcm5lZXQgS2F1ciIsImFsc29UZW5hbnRzIjpbImlkY3MtY2EiXSwic3ViX21hcHBpbmdhdHRyIjoidXNlck5hbWUiLCJwcmltVGVuYW50Ijp0cnVlLCJ0b2tfdHlwZSI6IkFUIiwiY2FfZ3VpZCI6ImNhY2N0LTA2Y2JlODcwODExMzQwY2E4ZGJlMWU4NTgwYWFkZmI2IiwiYXVkIjpbImh0dHBzOlwvXC9pZGNzLTlmZWIzZWY5MWNhNDQ0MWJiZWRhZGJjNTY0MGQ1ZmI4LmlkZW50aXR5Lm9yYWNsZWNsb3VkLmNvbTo0NDMiLCJ1cm46b3BjOmxiYWFzOmxvZ2ljYWxndWlkPWlkY3MtOWZlYjNlZjkxY2E0NDQxYmJlZGFkYmM1NjQwZDVmYjgiXSwidXNlcl9pZCI6IjJlNWIzZmE3MmI1YTQ2OTFiOGZiOTNkMDY0NzYwNmE3IiwiY2xpZW50QXBwUm9sZXMiOlsiQXV0aGVudGljYXRlZCBDbGllbnQiLCJJZGVudGl0eSBEb21haW4gQWRtaW5pc3RyYXRvciJdLCJ0ZW5hbnRfaXNzIjoiaHR0cHM6XC9cL2lkY3MtOWZlYjNlZjkxY2E0NDQxYmJlZGFkYmM1NjQwZDVmYjguaWRlbnRpdHkub3JhY2xlY2xvdWQuY29tOjQ0MyJ9.pS85ZctqAADIgBv2u8yjq8zZKjimsNyosontgJqUkSzx6H1Rd7mQqSRMb395_cJ7AHz4M7KRDNdOSZWmP9OwCFQzkYoeQNWGCJM0gKJxxWAvbqLp048Nl2Ch2MOq8oA3h2BfDxJ-NKH0iAeQrEMeh8G3Ki3g4fACbPbFsKDv5MdII4ey-0xxskkbU3zCfgN7iRvSPGDyb65zxbkGsYZan_NZg_yCdJPz7EME2p8o5jsNxjCANKxPnJQ9U88JFvkQq6RPTvsihgOI4CtBu12Faa71VuFWltza0dwtbI8z8QImTKYx84WQgWUtszQ_DKxYJvfCeEzAhsulUSeBvaE-Mg"
```
Comment (varmax2511) [inline examples/analytics/main.tf:52]: we shouldn't expose idcs token data in example

File: examples/analytics/main.tf [hunk 2]
```diff
@@ -69,41 +67,31 @@ resource "oci_analytics_analytics_instance" "test_oce_instance_public" {
...
-output "test" {
-  value = data.oci_analytics_analytics_instances.test_instance.id
+# Create a vanity url for the instance
+resource "oci_analytics_analytics_instances_vanity_url" "test_analytics_instances_vanity_url" {
+  #Required
+  analytics_instance_id = oci_analytics_analytics_instance.test_oce_instance_public.id
+  ca_certificate        = file("/Users/papakaur/Downloads/TestVanityUrls/RootCA.crt")
+  hosts                 = ["analyticsdev"]
+  private_key           = file("/Users/papakaur/Downloads/TestVanityUrls/analyticsdevCA.key")
+  public_certificate    = file("/Users/papakaur/Downloads/TestVanityUrls/analyticsdevCA.crt")
```
Comment (varmax2511) [inline examples/analytics/main.tf:90]: this shouldn't be exposed

Comment (govindrao55) [reply examples/analytics/main.tf]: addressed

File: examples/analytics/main.tf (hunk 1 above)
Comment (govindrao55) [reply examples/analytics/main.tf]: addressed

**Review by varmax2511 — APPROVED** [review]

**Pull request merged**